rust/workspace-crate/cookiecutter.json

```
@@ -8,6 +8,16 @@
   "repo_name": "{{ github_repo_name() }}",
   "crate_keywords": "blockchain serde",
   "crate_categories": "cryptography no-std",
+<<<<<<< HEAD
   "_extensions": ["local_extensions.GithubRepoNameExtension", "local_extensions.GithubUsernameExtension"]
+=======
+  "_extensions": ["local_extensions.GithubRepoNameExtension"],
+  "license": [
+    "Same as Workspace",
+    "Apache-2.0",
+    "AGPL-3.0-only",
+    "GPL-3.0-only"
+  ]
+>>>>>>> 4b90cd2 (Add license choice, use workspace.package.)
```

:x: looks like a bad merge
```suggestion
  "_extensions": ["local_extensions.GithubRepoNameExtension", "local_extensions.GithubUsernameExtension"],
  "license": [
    "Same as Workspace",
    "Apache-2.0",
    "AGPL-3.0-only",
    "GPL-3.0-only"
  ]
```

---

Hmm, that's pretty blech...